guide/guide.md

```diff
@@ -109,6 +109,30 @@ Put spaces before and after `as`:
 let cstr = "Hi\0" as *const str as *const [u8] as *const std::os::raw::c_char;
 ```
 
+### Type Aliases
+
+A type alias should follow normal line break rules. If one must break the line,
+then do it after the `=` first, and after the `type` next. Keep the type alias
+and `=` on the same line. Keep `pub` and `type` on the same line. If one must
+break, indent exactly once from the indentation of `type`.
+
+#### Single-line
+
+```rust
+[pub] type FooBar = Baz<i32>;
```

Comment:
Instead of using ad-hoc `[]` syntax, maybe we should just duplicate the example to be as concrete as possible:

```rust
type FooBar = Baz<i32>;
pub type FooBar = Baz<i32>;
```